components/Home.js

```diff
@@ -0,0 +1,22 @@
+import { View, StyleSheet } from 'react-native'
+import Header from './Header'
+import GreetingButton from './GreetingButton'
+
+const Home = () => {
+    return (
+        <View style={styles.homeContainer}>
+            <Header />
```

Comment:
Home component set-up is great so far. Can't wait to add state! 
This is where we're add hooks to update our state in RN.